src/js/edu-benefits/utils/validations.js

```diff
@@ -303,8 +303,8 @@ function isValidBenefitsHistoryPage(data) {
 }
 
 function isValidRotcScholarshipAmount(data) {
-  return isValidField(isValidMonetaryValue, data.amount)
-    && isValidField(isValidYear, data.year);
+  return (isBlank(data.amount.value) || isValidField(isValidMonetaryValue, data.amount))
+    && (isBlank(data.amount.value) || isValidField(isValidYear, data.year));
```

Comment:
Did you mean `isBlank(data.year.value)` here? Also, does it make sense for us to allow additional entries to be added if the values are blank?